tests/unit/multi_select_test.coffee

```diff
@@ -5,39 +7,43 @@ moduleForComponent 'multi-select', '[Unit] Multi select component',
     'template:select-item-layout'
     'template:select-item'
   ]
+  teardown: ->
+    Ember.run ->
+      multiSelect.destroy()
+    multiSelect = null
```

Comment:
How does this work? Wouldn't `multiSelect` be `null` when the run loop is called?